doc/stdlib/base.rst

```diff
@@ -833,54 +833,97 @@ Errors
 
    Throw an ``AssertionError`` if ``cond`` is false. Preferred syntax for writing assertions.
 
-.. data:: AssertionError
+.. function:: ArgumentError(msg)
+
+   The parameters to a function call do not match a valid signature.
+
+.. function:: AssertionError([msg])
 
    The asserted condition did not evalutate to ``true``.
 
-.. data:: ArgumentError
+.. function:: BoundsError([a],[i])
+
+   An indexing operation into an array, ``a``, tried to access an out-of-bounds element, ``i``.
+
+.. function:: DimensionMismatch([msg])
+
+   The objects called do not have matching dimensionality.
+
+.. function:: DivideError()
 
-   The parameters given to a function call are not valid.
+   Integer division was attempted with a denominator value of 0.
 
-.. data:: BoundsError
+.. function:: DomainError()
 
-   An indexing operation into an array tried to access an out-of-bounds element.
+   The arguments to a function or constructor are outside the valid domain.
 
-.. data:: EOFError
+.. function:: EOFError()
 
    No more data was available to read from a file or stream.
 
-.. data:: ErrorException
+.. function:: ErrorException(msg)
 
    Generic error type. The error message, in the `.msg` field, may provide more specific details.
 
-.. data:: KeyError
+.. function:: InexactError()
+
+   Type conversion cannot be done exactly.
+
+.. function:: InterruptException()
+
+   The process was stopped by a terminal interrupt (CTRL+C).
+
+.. function:: KeyError(key)
 
    An indexing operation into an ``Associative`` (``Dict``) or ``Set`` like object tried to access or delete a non-existent element.
 
-.. data:: LoadError
+.. function:: LoadError(file::AbstractString, line::Int, error)
 
    An error occurred while `including`, `requiring`, or `using` a file. The error specifics should be available in the `.error` field.
 
-.. data:: MethodError
+.. function:: MethodError(f, args)
 
    A method with the required type signature does not exist in the given generic function.
 
-.. data:: ParseError
+.. function:: NullException()
+
+   An attempted access to a ``Nullable`` with no defined value.
+
+.. function:: OutOfMemoryError()
+
+   A generated syntax tree was too large for the available memory.
```

Comment:
Yeah looks better now - some of the makefile dependencies are probably a little off for the doc building.

Last minor nit, I think "a wraparound" would be better than "an wraparound"